### DISCUSSION TITLE
Change Encoder Wheel mode speed?

### DISCUSSION THREAD

**Discussion by Lalaith2** [sstaub/eosstreamdeckplus#5] [Q&A]
Hello!

Just downloaded and got this set up and I couldn't be happier. Thank you so much for creating this and making it avaliable for free it will be life changing.

I'm curious if there is a way to shift the speed of each encoder wheel mode? I find myself needing to swap between Acc for adjusting focus and Fine for adjusting gobos. Maybe this is not possible but I figured I'd ask.

I'd also like to find out if it is at all concieveable to get the gobo select encoder to step through gobos instead of slide from one to the next. Again, perhaps not feasible or too complex to be worth it.

Thanks again!

**Comment by sstaub** [comment]
First can done in setup. The second is not possible by system.

**Reply by ST0N3BRIDGE** [reply]
Hi sstaub, thanks for creating this. Im using it for the first time in a show and having the same issue of pan/tilt encoders being too slow. Ive tried going in to setup and increasing the encoder labelled ENC ADJ ACCEL in the hopes it would speed up the scroll speed but it appears to make no difference. 
Could you advise?
many thanks
